.ansible-lint

```diff
@@ -37,5 +37,6 @@ exclude_paths:
   - tests/files/custom_cni/cilium.yaml
   - venv
   - .github
+  - docs/ansible/variable_reference.yml
```

Comment:
let's keep grouping the variables by domain in multiple files. 
then let's have people the cherry pick/copy past what they need:

cp docs/ansible/variables/cloud/gcp.yml  inventory/mycluster/group_vars/ 
cp docs/ansible/variables/network/calico.yml  inventory/mycluster/group_vars/

Reply:
Like, one file per role defaults ? That's mostly redundant with the defaults file themselves, no ?
Especially if we recreate the hierarchy.

Reply:
it's already redundant with roles.  

not all per role but some hierarchy especially when configuration are exclusive:
- calico vs weave vs cilium...
- crio vs docker vs .... 
- gcp vs azure ... 

and then maybe some hierarchy but not down to the role:
- kubernetes/vars.yml (all cluster configs)
- apps/vars.yaml  (all apps)

but can comes later too. need a bit more though on it.